src/bioregistry/data/bioregistry.json

```diff
@@ -74466,4 +74490,4 @@
     },
     "pattern": "^\\d+$"
   }
-}
\ No newline at end of file
+}
```

Comment:
sorry to say but you have to remove the trailing newline otherwise the lint check won't pass. Besides that, this is ready to merge

Reply:
wow... I thought we had left windows/linux/mac newline issues years behind us.